Tests/RxSwift/WeakTests.swift

```diff
@@ -133,4 +133,40 @@ class WeakTests: XCTestCase {
         let expected: [RxEvent: Int] = [.next: 0, .error: 0, .completed: 0, .disposed: 1]
         XCTAssertTrue(events == expected)
     }
+    
+    private class Wrapper<T> {
+        let value: T
+        
+        init(value: T) {
+            self.value = value
+        }
+    }
+    
+    private var wrapped: Wrapper<Int>? = nil
+    
+    func testFlatMap() {
```

Comment:
I think we're missing tests for your new subscribe overload ?

Reply:
Regarding that one, I was not sure of the testing strategies to have do the overloads.

Reply:
How do you mean exactly ?